sos/report/plugins/containerd.py

```diff
@@ -0,0 +1,33 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import (Plugin, PluginOpt, RedHatPlugin,
+                                UbuntuPlugin, CosPlugin)
+
+
+class Containerd(Plugin, RedHatPlugin, UbuntuPlugin, CosPlugin):
+
+    short_desc = 'Containerd containers'
+    plugin_name = 'containerd'
+    profiles = ('container',)
+
+    option_list = [
+        PluginOpt('logs', default=False,
+                  desc='collect stdout/stderr logs for containerd')
+    ]
+
+    def setup(self):
```

Comment:
Would collecting `containerd config dump` output be helpful here alongside the directory?